### ISSUE TITLE
Sonar Qube Latest version is not supported

### ISSUE THREAD

**Issue by savojha10** [fundacionjala/enforce-sonarqube-plugin#139]
Hi team , 

I was trying it with the latest version of Sonar 7.9.1. And it failed.  With some classes  which are now deprecated by SOnar and hence this code doesnt work anymore. 

Thanks 
S O

**Comment by NazareenaSulthana** [comment]
Yes I was trying with latest version of Sonar 7.9.1 and it is failing, any help is appreciated.
